### ISSUE TITLE
is it possible to export settings?

### ISSUE THREAD

**Issue by TD540** [muffinresearch/favicon-customizer#68]
because with every browser reset, I tend to have to reconfigure every icon again...

it would be nice if it got save/synced to my firefox account somehow...

**Comment by muffinresearch** [comment]
Yep, it should be possible to provide an export/import and or use storage sync.
